cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-bal-project.txt

```diff
@@ -2,11 +2,6 @@
 Compiling source
 	foo/winery:0.1.0
 
-Running Tests
```

Comment:
Don't we have this log at all or is it since we moved it to runtime?

Reply:
Since we moved the Running Tests portion to the runtime, it doesnt pick up this log which is why the tests were failing.